theme/less/gouvfr/community.less

```diff
@@ -124,152 +103,6 @@
         font-size: 13px;
     }
 
-    .reuses-list {
```

Comment:
Check if not used anywhere, or moved to `udata` core. If not, this must not be removed I think.